src/Dropdown/Dropdown.mdx

```diff
@@ -114,6 +115,7 @@ Easily display contextual overlays using custom trigger elements. Dropdown's pos
               </Dropdown>
             </Flex>
           </Button.Group>
+          <Input name="password" label="Password" />
```

Comment:
This is simply included to facilitate reviewer testing and will be removed. Ideally this would be tested via integration testing or VRT, and that's something to keep in mind moving forward.